@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -520,11 +525,27 @@ export class TextElementsRenderer {
     }
 
     get initialized(): boolean {
-        return this.m_initState === InitState.Initialized;
+        return this.m_initialized;
     }
 
     get initializing(): boolean {
-        return this.m_initState === InitState.Initializing;
+        return this.m_initPromise !== undefined;
```

Comment:
```suggestion
        return this.m_initPromise !== undefined && !this.initialized;
```

Reply:
Adding that condition there is unnecessary, initialized is always false when "this.m_initPromise !== undefined".

Reply:
Ok, I see now.